index.html

```diff
@@ -1362,9 +1362,7 @@ <h4>A basic notification with some additional detail</h4>
     </section>
 
     <section id="binary-fixity">
-      <h2>Binary Resource Fixity</h2>
```

Comment:
We still need this major heading I think

Reply:
what would Darth Vader say here

Reply:
@zimeon D'oh... yes, we should still have that.

Reply:
ok, I have restored the h2 and marked the other two headers in the section up as h3